service/src/main/java/uk/gov/hmcts/reform/fpl/model/Children.java

```diff
@@ -36,6 +37,8 @@ public Children(Child firstChild, Child... additionalChildren) {
     }
 
     @JsonIgnore
+    @Valid
```

Comment:
Is that right place?

Reply:
It can go in either. Let's move it closer to the child for better clarity on which is being validated.